python/cudf/cudf/_lib/pylibcudf/table.pyx

```diff
@@ -13,24 +13,41 @@ from .column cimport Column
 
 
 cdef class Table:
-    """A set of columns of the same size."""
-    def __init__(self, object columns):
+    """A set of columns of the same size.
```

Comment:
```suggestion
    """
    A list of columns of the same size.
```
Or collection (the columns need not be unique, and are ordered...)

Reply:
For now it's typed to require a list. We can see about relaxing that in the future.